src/index.js

```diff
@@ -1181,6 +1181,7 @@ Carousel.defaultProps = {
   framePadding: '0px',
   height: 'auto',
   heightMode: 'max',
+  onDragStart() {},
```

Comment:
Added default prop to avoid guards everywhere of `if (this.props.onDragStart) ...`

Reply:
thanks!